src/test/java/test/SplashScreenKeTest.java

```diff
@@ -0,0 +1,25 @@
+package test;
+
+import org.testng.annotations.Test;
+import pages.SplashScreenKePage;
+import providers.KeSplashScreenStringProviders;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class SplashScreenKeTest extends TestBase {
+    KeSplashScreenStringProviders keSplashScreenStringProviders = new KeSplashScreenStringProviders();
+
+    @Test
+    public void testSplashScreen() {
+        SplashScreenKePage splashScreenKePage = new SplashScreenKePage(driver);
+        assertThat(splashScreenKePage.getTitleText().getText(), is(keSplashScreenStringProviders.getTitleString()));
+        System.out.println("\nThe title is : " + splashScreenKePage.getTitleText().getText());
```

Comment:
Why do you need this?

Reply:
Removed it for all places `sout` was. We were printing it to console for testing.